gr-digital/python/digital/qa_constellation_receiver.py

```diff
@@ -39,7 +39,7 @@
 TIMING_OFFSET = 1.0
 
 # RECEIVER PARAMETERS
-FREQ_BW = 2 * math.pi / 100.0
+FREQ_BW = 2.3e-3
```

Comment:
To the old bandwidth of `2 * pi / 100` a new bandwidth of `2.3e-3` corresponds approximately.